lang/en/settings.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$lang['cache'] = 'Tick to disable the cache';
```

Comment:
I think "Disable the cache" would be enough, and more consistent with the other options of the Dokuwiki admin panel